src/lib/pages/podcast-episodes/api/load-podcast-from-url.ts

```diff
@@ -3,7 +3,12 @@ import { podcastGet } from '$lib/entities/podcast';
 
 export const loadPodcastFromURL: Load = async ({ params }) => {
   try {
-    const podcast = await podcastGet(parseInt(params.id, 10));
+    const integerID = parseInt(params.id, 10);
+    if (isNaN(integerID)) {
+      throw integerID;
```

Comment:
`throw`ing `NaN` feels a bit lousy, tbh)
Wouldn't a 404 error be more appropiate?

Reply:
Idk, I just figured that if we were to return 404s, we'd better design a decent 404 experience, but in this way we just redirect to Podcast Gallery